ocs_ci/utility/lvmo_utils.py

```diff
@@ -0,0 +1,145 @@
+
+    """
+    vg_manager = get_pods_having_label(label, namespace)
+    return Pod(**vg_manager[0])
+
+
+def get_lvm_controller_manager_pod(
+    label=constants.LVMO_POD_LABEL["411"]["controller_manager_label"],
+    namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+):
+    """
+    Get controller manager pod in the lvm cluster
+
+    Args:
+        label (str): Label associated with controller manager pod
+        namespace (str): Namespace in which controler manager pod is residing
+
+    Returns:
+        Pod: Pod object of controller manager pod
+
+    """
+    lvm_controller = get_pods_having_label(label, namespace)
+    return Pod(**lvm_controller[0])
+
+
+def get_topolvm_controller_pod(
+    label=constants.LVMO_POD_LABEL["411"]["topolvm-controller_label"],
+    namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+):
+    """
+    Get topolvm controller pod in the lvm cluster
+
+    Args:
+        label (str): Label associated with topolvm controller pod
+        namespace (str): Namespace in which topolvm controler pod is residing
+
+    Returns:
+        Pod: Pod object of topolvm controller pod
+
+    """
+    topolvm_controller = get_pods_having_label(label, namespace)
+    return Pod(**topolvm_controller[0])
+
+
+def get_topolvm_node_pod(
+    label=constants.LVMO_POD_LABEL["411"]["topolvm-node_label"],
+    namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+):
+    """
+    Get topolvm node pod in the lvm cluster
+
+    Args:
+        label (str): Label associated with topolvm node pod
+        namespace (str): Namespace in which topolvm node pod is residing
+
+    Returns:
+        Pod: Pod object of topolvm node pod
+
+    """
+    topolvm_node_pod = get_pods_having_label(label, namespace)
+    return Pod(**topolvm_node_pod[0])
+
+
+def lvmo_health_check_base():
+    """
+    Check health of lvm cluster by checking the following:
+    lvm cluster cr status,
+    lvm pods status
+
+    Returns:
+        bool: True if all checks passed, raise exception otherwise
+    """
+    oc_obj = OCP(
+        namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+        kind="lvmcluster",
+        resource_name="lvmcluster",
+    )
+    lvmcluster_status = oc_obj.get("lvmcluster")
+    if not (lvmcluster_status["status"]["ready"]):
+        log.info("lvmcluster is not ready")
+        raise LVMOHealthException(
+            f"LVM Cluster status is {lvmcluster_status['status']['ready']}"
+        )
+
+    pods_status = list()
+    vg_manager_status = get_lvm_vg_manager_pod().get("status")["status"]["phase"]
+    pods_status.append(vg_manager_status)
+    topolvm_node_status = get_topolvm_node_pod().get("status")["status"]["phase"]
+    pods_status.append(topolvm_node_status)
+    controller_manager_status = get_lvm_controller_manager_pod().get("status")[
+        "status"
+    ]["phase"]
+    pods_status.append(controller_manager_status)
+    lvm_manager_status = get_lvm_vg_manager_pod().get("status")["status"]["phase"]
```

Comment:
Is this the same as  line 110?

Reply:
shouldnt be, will fix it, thanks for noticing